src/components/JobLogsContainer/JobLogsContainer.js

```diff
@@ -85,9 +85,9 @@ const JobLogsContainer = props => {
 
   const getFileNameField = record => {
     const fileName = get(record.exportedFiles, '0.fileName');
-    const isSuccessfulRecord = record.status === JOB_EXECUTION_STATUSES.SUCCESS;
+    const areAllJobsFailed = !record.progress.exported;
 
-    if (!isSuccessfulRecord) return fileName;
+    if (areAllJobsFailed) return fileName;
```

Comment:
@Vladyslav-Velytskyi @VictorSoroka1 I wonder if we really need to add an extra variable or just can use

`!record.progress.exported;`

As for me this condition is short and clear enough: If not exported - return fileName

What do you think?

Reply:
Done, thanks, nice suggestion